taipy/core/config/checkers/_scenario_config_checker.py

```diff
@@ -38,10 +38,33 @@ def _check(self) -> IssueCollector:
                 self._check_addition_data_node_configs(scenario_config_id, scenario_config)
                 self._check_additional_dns_not_overlapping_tasks_dns(scenario_config_id, scenario_config)
                 self._check_tasks_in_sequences_exist_in_scenario_tasks(scenario_config_id, scenario_config)
+                self._check_if_children_config_id_is_overlapping_with_properties(scenario_config_id, scenario_config)
                 self._check_comparators(scenario_config_id, scenario_config)
 
         return self._collector
 
+    def _check_if_children_config_id_is_overlapping_with_properties(
+        self, scenario_config_id: str, scenario_config: ScenarioConfig
+    ):
+        if scenario_config.tasks:
+            for task in scenario_config.tasks:
+                if isinstance(task, TaskConfig) and task.id in scenario_config.properties:
```

Comment:
I understand why we want to have `isinstance` here but is it really necessary?

Reply:
Yes, because if the `task` is not a TaskConfig, the `task.id` will fail and raise an AttributeError.
In the checker, we only want to log the error, not raise anything directly here.

Reply:
in that case, should the log.error show a separate message to make it more concise?

Reply:
The case where the `task` is not TaskConfig is handle by `_check_task_configs()` method, so there is a error log their.

Reply:
hmm yeah okay then :D